src/kokkos/ekat_kokkos_types.hpp

```diff
@@ -1,16 +1,36 @@
 #ifndef EKAT_KOKKOS_TYPES_HPP
 #define EKAT_KOKKOS_TYPES_HPP
 
-#include "ekat/ekat_config.h"
-#include "ekat/kokkos/ekat_kokkos_meta.hpp"
-#include "ekat/std_meta/ekat_std_type_traits.hpp"
+#include "ekat_kokkos_meta.hpp"
+#include "ekat_std_type_traits.hpp"
 
 /*
  * Header contains globally useful kokkos-related types for ekat.
  */
 
 namespace ekat {
 
+template <typename ExeSpace>
```

Comment:
This used to be in ekat_arch, but that is now in Core. I figured kokkos types was an ok place, since it's likely to be included by whoever uses ekat's kokkos utils.